src/cargo/core/profiles.rs

```diff
@@ -776,6 +789,28 @@ impl fmt::Display for PanicStrategy {
     }
 }
 
+/// The setting for choosing which symbols to strip
+#[derive(Clone, Copy, PartialEq, Eq, Debug, Hash, PartialOrd, Ord, serde::Serialize)]
+#[serde(rename_all = "lowercase")]
+pub enum Strip {
+    /// Only strip debugging symbols
+    DebugInfo,
+    /// Don't remove any symbols
+    None,
+    /// Strip all non-exported symbols from the final binary
+    Symbols,
+}
+
+impl fmt::Display for Strip {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match *self {
+            Strip::DebugInfo => "unwind",
+            Strip::None => "abort",
+            Strip::Symbols => "symbols",
```

Comment:
Seems like there's a typo here for `Strip`'s `Display` implementation

Reply:
Thanks, I've missed that!